applications/MappingApplication/custom_python/mapping_python_application.cpp

```diff
@@ -25,27 +25,23 @@
 #include "custom_python/add_custom_mappers_to_python.h"
 
 
-namespace Kratos
-{
-
-namespace Python
-{
+namespace Kratos {
+namespace Python {
 
-using namespace pybind11;
+namespace py = pybind11;
```

Comment:
are u sure it is a good idea to define this outside of the function? it could make a mess with cotire...

Reply:
You mean outside of `AddCustomMappersToPython`? Bcs I also want to use pybindstuff outside of this function

But if you say it makes problems, then I can move it inside and use `pybind11::` outside